src/components/footer/footer.component.jsx

```diff
@@ -17,40 +20,123 @@ export const Footer = () => {
           </div>
           <div>
             <ul className={classes.list}>
-              <li className={classes.listItem}>Hermez</li>
-              <li className={classes.listItem}>Hermez Network</li>
-              <li className={classes.listItem}>Hermez Wallet</li>
-              <li className={classes.listItem}>Network explorer</li>
+              <li className={`${classes.listItem} ${classes.listItemHead}`}>
+                Hermez
+              </li>
+              <li className={classes.listItem}>
+                <Link to={routes.find(el => el.label === 'Payments network').to}>
```

Comment:
I don't think that finding routes by its label is reliable... They are likely to change. I would add an identifier to the routes objects with an enum, so then you can find them by this enum and identifier.

Reply:
I added `id`s to routes and find now goes for the id.

Reply:
But it seems to me this is not what you had in mind? :) Let me know.

Reply:
Yes, that was the idea!